packages/rum-core/src/domain/action/trackClickActions.ts

```diff
@@ -315,23 +314,21 @@ function newClick(
       }
 
       const { resourceCount, errorCount, longTaskCount } = eventCountsSubscription.eventCounts
-      const clickAction: ClickAction = assign(
-        {
-          type: ActionType.CLICK as const,
```

Comment:
`'type' is specified more than once, so this usage will be overwritten. ts(2783)`

spreading `clickActionBase` will always overwrites `type`